Misc/NEWS.d/next/Library/2025-02-07-00-48-07.gh-issue-112632.95MM0C.rst

```diff
@@ -0,0 +1 @@
+Adds a `block_style` boolean parameter to :func:`pprint.pp` which inserts a newline after opening parentheses and braces of printed objects and indents their contents. In other words, enabling this option results in formatted output looking similar to the block style formatting from :func:`json.dumps` when `indent` is set to a positive integer. Contributed by Stefan Todoran.
```

Comment:
try something like (don't forget about grammar and line length):
```rst
Add a *block_style* keyword argument for :func:`pprint.pprint` ... Contributed
by Stefan Todoran.
```

You can see other examples in the same folder